test/RelayClient.test.ts

```diff
@@ -189,12 +177,12 @@ describe('RelayClient', function () {
             ...FAKE_RELAY_REQUEST.request,
             relayHub: expectedRelayHubAddress,
           },
-        }, FAKE_REQUEST_CONFIG);
+        });
 
         expect(actualRelayHubAddress).to.equal(expectedRelayHubAddress);
       });
 
-      it('should return tx request with fees receiver in request data from given hub info if no fees receiver given in request', async function () {
+      it.skip('should return tx request with fees receiver in request data from given hub info if no fees receiver given in request', async function () {
```

Comment:
How do we plan to fix the skipped test? Do we want to add a specific task for all of them?

Reply:
now I forgot to discuss it with you, maybe after the daily we can check it.

Reply:
updated